components/taggroup/stories/taggroup.stories.js

```diff
@@ -74,16 +76,16 @@ Default.args = {
 
 // ********* DOCS ONLY ********* //
 /**
- * When horizontal space is limited in a tag group, the individual tags wrap to form another line.
+ * A tag group can contain removable tags when the context is for editing, or non-removable tags when tags are read-only, but removable and non-removable tags cannot be combined within the tag group.
  */
-export const OverflowItems = Template.bind({});
-OverflowItems.tags = ["autodocs", "!dev"];
-OverflowItems.parameters = {
+export const Removable = Template.bind({});
```

Comment:
I changed the `OverflowItems` story to `Removable` because it seemed more like a variant than `OverflowItems`, and it aligns with what was on the docs site.